proyecto-compensations/force-app/main/default/aura/CompensationsSelect/CompensationsSelectController.js

```diff
@@ -0,0 +1,9 @@
+({
+  onChange: function(cmp, evt, helper) {
+    var event = $A.get("e.c:TypeSelectionEvent");
+      console.log(cmp.find("select").get("v.value"));
```

Comment:
its a good practice to remove the logs from the files before uploading the files to the repository.